web/www/missa/Latin/Tempora/Quad6-3.txt

```diff
@@ -112,7 +116,12 @@ quóniam adhuc hoc, quod scriptum est, oportet impléri in me: Et cum~
 iníquis deputátus est. Etenim ea, quæ sunt de me, finem habent. C. At~
 illi dixérunt: S. Dómine, ecce duo gládii hic. C. At ille dixit eis: J.~
 Satis est. C. Et egréssus ibat secúndum consuetúdinem in montem~
-Olivárum. Secúti sunt autem illum et discípuli. Et cum pervenísset ad~
+Olivárum.
+(sed rubrica 1955 aut rubrica 1960 haec versus omittuntur)
+
+(rubrica 1955 aut rubrica 1960 dicitur)
+In illo témpore: Egréssus ibat secúndum consuetúdinem in montem Olivárum.~
```

Comment:
Sorry, fixed. Due to the wording nuances I would leave fixing this in other languages to respective authors.